<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChequePrinter</class>
 <widget class="QMainWindow" name="ChequePrinter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cheque Printer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Cheque Details</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="cmdNewCheque">
            <property name="text">
             <string>New Cheque</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cmdSave">
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cmdFillAmountInWords">
            <property name="text">
             <string>Fill Amount In Words</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="cmdDelete">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cmdDeleteAll">
            <property name="text">
             <string>Delete All</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tblCheques"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Cheque Preview</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Profile:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cboProfile"/>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QFrame" name="frame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>2</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="vLay"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>683</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport_Suppliers_From_Excel"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="actionExport_PDF"/>
    <addaction name="actionSend_PDF_via_Email"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPrintout_Settings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Cheque_Printer"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionSinge_Page"/>
    <addaction name="actionFacing_Pages"/>
    <addaction name="actionAll_Pages"/>
    <addaction name="separator"/>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="separator"/>
    <addaction name="actionFit_Whole_Page"/>
    <addaction name="actionFit_Page_Width"/>
    <addaction name="separator"/>
    <addaction name="actionFirst_Page"/>
    <addaction name="actionPrevious_Page"/>
    <addaction name="actionNext_Page"/>
    <addaction name="actionLast_Page"/>
    <addaction name="separator"/>
    <addaction name="actionPage_Setup"/>
    <addaction name="actionPrint"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Main ToolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionImport_Suppliers_From_Excel"/>
   <addaction name="actionExport_PDF"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Printing Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionFetch_Quickbooks_Suppliers">
   <property name="text">
    <string>Fetch Quickbooks Suppliers</string>
   </property>
  </action>
  <action name="actionImport_Suppliers_From_Excel">
   <property name="icon">
    <iconset resource="cp_res.qrc">
     <normaloff>:/cp_images/MS-Office-2003-Excel-icon.png</normaloff>:/cp_images/MS-Office-2003-Excel-icon.png</iconset>
   </property>
   <property name="text">
    <string>Load Excel File</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="cp_res.qrc">
     <normaloff>:/cp_images/Impresora-icon.png</normaloff>:/cp_images/Impresora-icon.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionExport_PDF">
   <property name="icon">
    <iconset resource="cp_res.qrc">
     <normaloff>:/cp_images/PDF-icon.png</normaloff>:/cp_images/PDF-icon.png</iconset>
   </property>
   <property name="text">
    <string>Export PDF</string>
   </property>
  </action>
  <action name="actionSend_PDF_via_Email">
   <property name="text">
    <string>Send PDF via Email</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="cp_res.qrc">
     <normaloff>:/cp_images/Button-Close-icon.png</normaloff>:/cp_images/Button-Close-icon.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionPrintout_Settings">
   <property name="text">
    <string>Printout Settings</string>
   </property>
  </action>
  <action name="actionAbout_Cheque_Printer">
   <property name="text">
    <string>About Cheque Printer</string>
   </property>
  </action>
  <action name="actionSinge_Page">
   <property name="icon">
    <iconset resource="cp_res.qrc">
     <normaloff>:/cp_images/view-page-one-32.png</normaloff>:/cp_images/view-page-one-32.png</iconset>
   </property>
   <property name="text">
    <string>Singe Page</string>
   </property>
  </action>
  <action name="actionFacing_Pages">
   <property name="icon">
    <iconset resource="cp_res.qrc">
     <normaloff>:/cp_images/view-page-sided-32.png</normaloff>:/cp_images/view-page-sided-32.png</iconset>
   </property>
   <property name="text">
    <string>Facing Pages</string>
   </property>
  </action>
  <action name="actionAll_Pages">
   <property name="icon">
    <iconset resource="cp_res.qrc">
     <normaloff>:/cp_images/view-page-multi-32.png</normaloff>:/cp_images/view-page-multi-32.png</iconset>
   </property>
   <property name="text">
    <string>All Pages</string>
   </property>
  </action>
  <action name="actionPage_Setup">
   <property name="icon">
    <iconset resource="cp_res.qrc">
     <normaloff>:/cp_images/page-setup-32.png</normaloff>:/cp_images/page-setup-32.png</iconset>
   </property>
   <property name="text">
    <string>Page Setup</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset resource="cp_res.qrc">
     <normaloff>:/cp_images/zoom-in-32.png</normaloff>:/cp_images/zoom-in-32.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset resource="cp_res.qrc">
     <normaloff>:/cp_images/zoom-out-32.png</normaloff>:/cp_images/zoom-out-32.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionFit_Whole_Page">
   <property name="icon">
    <iconset resource="cp_res.qrc">
     <normaloff>:/cp_images/fit-page-32.png</normaloff>:/cp_images/fit-page-32.png</iconset>
   </property>
   <property name="text">
    <string>Fit Whole Page</string>
   </property>
  </action>
  <action name="actionFit_Page_Width">
   <property name="icon">
    <iconset resource="cp_res.qrc">
     <normaloff>:/cp_images/fit-width-32.png</normaloff>:/cp_images/fit-width-32.png</iconset>
   </property>
   <property name="text">
    <string>Fit Page Width</string>
   </property>
  </action>
  <action name="actionFirst_Page">
   <property name="icon">
    <iconset resource="cp_res.qrc">
     <normaloff>:/cp_images/go-first-32.png</normaloff>:/cp_images/go-first-32.png</iconset>
   </property>
   <property name="text">
    <string>First Page</string>
   </property>
  </action>
  <action name="actionPrevious_Page">
   <property name="icon">
    <iconset resource="cp_res.qrc">
     <normaloff>:/cp_images/go-previous-32.png</normaloff>:/cp_images/go-previous-32.png</iconset>
   </property>
   <property name="text">
    <string>Previous Page</string>
   </property>
  </action>
  <action name="actionNext_Page">
   <property name="icon">
    <iconset resource="cp_res.qrc">
     <normaloff>:/cp_images/go-next-32.png</normaloff>:/cp_images/go-next-32.png</iconset>
   </property>
   <property name="text">
    <string>Next Page</string>
   </property>
  </action>
  <action name="actionLast_Page">
   <property name="icon">
    <iconset resource="cp_res.qrc">
     <normaloff>:/cp_images/go-last-32.png</normaloff>:/cp_images/go-last-32.png</iconset>
   </property>
   <property name="text">
    <string>Last Page</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="cp_res.qrc"/>
 </resources>
 <connections/>
</ui>
